master.cfg

```diff
@@ -347,11 +347,16 @@ for builder in master_config["builders"]:
 
     # Add builder only if it's not a protected branches one
     if builder not in github_status_builders:
+        tags = [os_name]
+        if arch == "s390x" and builder in builders_galera_mtr:
+            tags += ["experimental"]
+        if "sid" in builder or "stream-9" in builder:
+            tags += ["bleeding-edge"]
```

Comment:
@grooverdan 
Do we need also to filter out bleeding-edge?
Or do we need a dedicated filter for it?

Reply:
probably doesn't need to be dedicated filter.  Happy to have a tag there as "just FYI" for the reader at the moment.